setup.py

```diff
@@ -40,4 +44,5 @@ def read(*parts):
         maintainer='Twisted Matrix Labs Developers',
         version=versioneer.get_version(),
         cmdclass=versioneer.get_cmdclass(),
+        python_requires=">= 3.8",
```

Comment:
I t
I think that we also need to remove the universal build

The resulting wheel is

```
constantly-0+untagged.1.g6bbe661-py2.py3-none-any.whl
```